### PR TITLE
Add coordinator layout

### PR DESCRIPTION
**Pull request by YazanKittaneh**
Added CoordinatorLayout.
Removed article detail from the PR.

### CODE REVIEW

**Comment by YazanKittaneh** [comment]
after
before

**Comment by YazanKittaneh** [comment]
current app status

File: .idea/gradle.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <component name="GradleSettings">
     <option name="linkedExternalProjectsSettings">
       <GradleProjectSettings>
-        <option name="distributionType" value="DEFAULT_WRAPPED" />
+        <option name="distributionType" value="LOCAL" />
```
Comment (LarryAsante) [inline .idea/gradle.xml]: Please keep the gradle distribution type as `DEFAULT_WRAPPED`

File: app/build.gradle [hunk 2]
```diff
@@ -29,14 +29,18 @@ android {
...
+    compile 'com.android.support:support-v4:24.2.1'
+    compile 'com.android.support:appcompat-v7:24.2.1'
+    compile 'com.android.support:design:24.2.1'
     compile 'com.jakewharton:butterknife:7.0.1'
     compile 'com.jakewharton.timber:timber:4.1.2'
     compile 'com.android.support:support-v4:24.2.1'
     compile 'com.squareup.retrofit2:retrofit:2.1.0'
     compile 'com.squareup.retrofit2:converter-gson:2.1.0'
+    testCompile 'junit:junit:4.12'
+    testCompile 'org.mockito:mockito-core:1.9.5'
```
Comment (LarryAsante) [inline app/build.gradle:42]: Nit-picking here: Make sure your changes really reflect what you said you're doing. Moving the mockito  and junit dependency around are not particularly the relevant changes here. This is distracting to anyone reviewing the code :)

File: app/src/main/java/edu/grinnell/grinnell_publications_android/Activities/MainActivity.java [hunk 3]
```diff
@@ -34,6 +37,7 @@
     @Bind(R.id.navigation_view) NavigationView navigationView;
```
Comment (LarryAsante) [inline app/src/main/java/edu/grinnell/grinnell_publications_android/Activities/MainActivity.java:37]: Nit-picking here again: How about we make all our member variables in this format : eg. `NavigationView nagivationView` will now become `NavigationView mNavigationView`. This way, we can easily differentiate b/n member variable and non-member variables.

File: app/src/main/res/layout/activity_main.xml [hunk 4]
```diff
@@ -7,10 +7,13 @@
     android:fitsSystemWindows="true"
     tools:context=".Activities.MainActivity">
 
-    <LinearLayout
-        android:layout_height="match_parent"
+    <android.support.design.widget.CoordinatorLayout
+        android:id="@+id/main_content"
```
Comment (LarryAsante) [inline app/src/main/res/layout/activity_main.xml:11]: Nice! I like that you keep the naming syntax here!

**Review by LarryAsante — CHANGES_REQUESTED** [review]
Nice start, @YazanKittaneh ! A few nit-picking comments but it all looks good. Kindly review them and make the changes and we should be good to go!

**Pull request closed without merging**